extensions/wikia/JSVariables/JSVariables.php

```diff
@@ -78,7 +78,12 @@ function wfJSVariablesTopScripts(Array &$vars, &$scripts) {
 
 	$vars['wgTransactionContext'] = Transaction::getAttributes();
 
-	$scripts .= Html::inlineScript("var wgNow = new Date();") .	"\n";
+	$scripts .= Html::inlineScript("var wgNow = new Date();") . "\n";
+
+	// ADEN-6676: Inline InstantGlobals to prevent AdBlock block
+	$instantGlobalsModule = new InstantGlobalsModule();
+	$instantGlobals = $instantGlobalsModule->getScript(new ResourceLoaderContext(new ResourceLoader(), $wg->Request));
```

Comment:
Could you please refactor a bit this method? (extract?) https://github.com/Wikia/app/blob/dev/extensions/wikia/InstantGlobals/InstantGlobalsModule.class.php#L48
So we don't need to pass ResourceLoaderContext. It seems not needed in this case.

Reply:
`InstantGlobalsModule::getVariablesValues` isn't cached. It means ~50 memcache calls for separate variables on every pageview. Maybe we could apply very short cache on top of it?

Reply:
@fraszczakszymon Indeed. Added default null parameter.
@kvas-damian The whole page is cached for 24 hours, do we need extra cache only for Instant Globals? But on the other hand why not, added Memcached usage.